Documentation/D2.md

```diff
@@ -21,17 +21,30 @@ During our first meeting we discovered that:
+  * Procedure: We will be preforming a retroactive analysis of the events that occure. Step one is to preform 30 valid actions. We will then pull the github logs timestamps and the RPis logs timestamps and compute the average difference to determine action laitency.
+  * Expected Results: We expect the results to be less than two seconds from the time a user preforms a valid action to when the device responds.
   * Independent Variables:
     * Git is a web service that relies on https protocols and tcp/ip packets through dns servers and our times to connect to those services could be affected by internet volume, security checks, and availability of the services. While it may be possible to call the GitHub servers directly to bypass the dns lookup, this should not significantly effect the time to communicate with the services.
+    * Repository size and commit size could be a potential factor. When pull requests are accepted they are merged which in large repositories and commits could take longer to complete. Because we are not interested in the performance of githubs algorythms we will be limiting the number of lines needed to merge to no more than 20 line changes.
 * Cost and marketability (A survey with standard questions that contribute to the goal of obtaining the level of interest, user acceptance, and device pricing)
+  * Experiment: We will attempt to determine the end user marketability of our device by surveying a population of developers familure with github.
+  * Procedure: Using an anonymous written(digitial or paper) survey with questions pertaining to user acceptance of the device using a veriety of dichotomous and Likert scales. The Likert scales will be built using current standards according to reasearch using a 5 point scale with the polor lables being delight and frustration.
+  * Expected Results: We expect the results to revieal that users will be more motivated to preform valid actions, and find programming more enjoyable or whimsicle.
   * Independent Variables:
     * The survey will not be peer reviewed and thus subject to a degree of bias. We will however take our best efforts not to ask leading questions and ask the questions in an unbiased order.
     * The survey sample size will likely be small and could contain sampling bias. While we will attempt to survey a variety of professionals, our access to developers within Colorado State University's student body is much more readily available.
 * Power consumption (Using standard RPi power statistics to estimate power consumption)
+  * Experiment: We will measure the power consumption of the device during a standard american work period of 9 hours (8 hours of working time + an hour lunch break).
+  * Procedure:
+  * Expected Results:
```

Comment:
We expect the device to consume 1 watt of power on average because the Raspberry Pi Zero with Wifi draws 0.7W on average. Powering the LEDs and the motor power bursts should average out to be a total power consumption of 1 watt.